OpenEdXMobile/src/main/java/org/edx/mobile/util/SecurityUtil.java

```diff
@@ -55,7 +61,7 @@ public static void clearUserData(@NonNull Context context) {
             }
         }
 
-        // Now clear the shared preferences
-        PrefManager.nukeSharedPreferences();
+        // Now clear all the shared preferences, ignore APP_INFO preferences
```

Comment:
`Now clear all the shared preferences except app related preferences` sounds like a better comment instead.